Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -96,24 +91,21 @@ struct ScheduleTransferSummaryData: ScheduleTransferSectionData {
     init(transfer: HyperwalletTransfer) {
         if let destinationAmount = transfer.destinationAmount,
             let destinationCurrency = transfer.destinationCurrency {
-            let destinationAmountFormattedString = destinationAmount.format(with: destinationCurrency)
-            guard let feeAmount = transfer.destinationFeeAmount else {
+            guard let feeAmount = transfer.destinationFeeAmount, Double(feeAmount) != 0 else {
                 rows.append((title: "transfer_amount_confirmation".localized(),
-                             value: destinationAmountFormattedString))
-                rows.append((title: "transfer_net_amount_confirmation".localized(),
-                             value: destinationAmountFormattedString))
+                             value: destinationAmount))
                 return
             }
             let transferAmountFormattedDouble = destinationAmount.formatToDouble(with: destinationCurrency)
             let feeAmountFormattedDouble = feeAmount.formatToDouble(with: destinationCurrency)
             let grossTransferAmount = transferAmountFormattedDouble + feeAmountFormattedDouble
 
             rows.append((title: "transfer_amount_confirmation".localized(),
-                         value: String(grossTransferAmount).format(with: destinationCurrency)))
+                         value: String(grossTransferAmount)))
```

Comment:
grossAmount still needs to be formatted as it's calculated and not returned from API. So add ` String(grossTransferAmount).format(with: destinationCurrency) `